package exception_handling;

//public class ubreachableblock {
/*
	public static void main(String[] args) {
		// TODO Auto-generated method stub
		System.out.println("program end");
		int[] num = new int[3];
		try {
			System.out.println(num[3]);
		} catch (Throwable e) { // PARENTCLASS BECAUSE
			System.out.println("Exception found and handle");
		} catch (ArithmeticException e) {
			System.out.println("Exception found and handle");
		} catch (ArrayIndexOutOfBoundsException e) { // UREACHABLE BLOCK
			System.out.println("Exception found and handle");
		} catch (Exception e) {
			System.out.println("Exception found and handle");
		}
		System.out.println("program end");

	}

}
*/